daemon/options/bpf.go

```diff
@@ -41,7 +41,7 @@ func (c *BpfConfig) AttachFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&c.Mode, "mode", "dual-engine", "controller plane mode, valid values are [kernel-native, dual-engine]")
 	cmd.PersistentFlags().BoolVar(&c.EnableMda, "enable-mda", false, "enable mda")
 	cmd.PersistentFlags().BoolVar(&c.EnableBpfLog, "enable-bpf-log", false, "enable ebpf log in daemon process")
-	cmd.PersistentFlags().BoolVar(&c.EnableAccesslog, "enable-accesslog", false, "enable accesslog in daemon process")
+	cmd.PersistentFlags().BoolVar(&c.EnableMonitoring, "monitoring", false, "enable kmesh traffic monitoring in daemon process")
```

Comment:
should we make it on by default?

If EnableMonitoring=false and EnableBpfLog = true, that's not valid, right?

We should validate them

Reply:
`EnableMonitoring` and `EnableBpfLog` are not related.
`EnableMonitoring` control Kmesh monitoring. And I will add a function to control accesslog in `kmeshctl`.
```sh
# Enable Kmesh's accesslog:
kmeshctl monitoring <kmesh-daemon-pod> --accesslog enable

# Disable Kmesh's accesslog:
kmeshctl monitoring <kmesh-daemon-pod> --accesslog disable

# Enable Kmesh's metrics and accesslog:
kmeshctl monitoring <kmesh-daemon-pod> --all enable

# Disable Kmesh's metrics and accesslog:
kmeshctl monitoring <kmesh-daemon-pod> --all disable
```